src/Vault.sol

```diff
@@ -74,25 +74,30 @@ contract Vault is Ownable {
 
         collateralDecimals = collateral.decimals();
         debtDecimals = debt.decimals();
-
     }
 
+    /*/////////////////////////////////////////////////////////////////////////
```

Comment:
this is solmate style (which I have proposed they change).

these don't work with code folding.  we do not use this style at Yield.

we havent decided on a style yet so feel free to be creative and try somethign new.  just make sure it:
a) doesn't affect the doc generation or etherscan natspec display
b) doesn't completely disappear on code folding

Reply:
Can i just say, nice use of 'je ne sais quoi'.
Will update to follow your proposed style, it makes sense.